lesson_04/main_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_main(t *testing.T) {
+	tests := []struct {
```

Comment:
I guess, we don't need that test case, usually there is no need to test main function at unit tests stage=)
But, no problem, it will works, so I approve that request.

Also, maybe we don't need to run `for` if we have only one statement to check.